pkg/fakerp/admin_handlers.go

```diff
@@ -173,3 +175,29 @@ func (s *Server) handleForceUpdate(w http.ResponseWriter, req *http.Request) {
 	}
 	s.log.Info("force-updated cluster")
 }
+
+// handleRunCommand handles running generic commands on a given vm within a scaleset in the cluster
+func (s *Server) handleRunCommand(w http.ResponseWriter, req *http.Request) {
+	cs := s.read()
+	if cs == nil {
+		s.internalError(w, "Failed to read the internal config")
+		return
+	}
+
+	hostname := chi.URLParam(req, "hostname")
+	command := chi.URLParam(req, "command")
+
+	ctx, err := enrichContext(context.Background())
+	if err != nil {
+		s.internalError(w, fmt.Sprintf("Failed to enrich context: %v", err))
+		return
+	}
+
+	err = s.plugin.RunCommand(ctx, cs, hostname, api.Command(command))
+	if err != nil {
+		s.internalError(w, fmt.Sprintf("Failed to run command: %v", err))
+		return
+	}
+
+	s.log.Infof("ran command on %s", hostname)
```

Comment:
ran _what_ command?

Reply:
also: would it be useful to at least log how long the command took _in lieu_ of actual metrics for now?

Reply:
both fixed, thanks - command timing can be seen from log timestamps